entrypoint.sh

```diff
@@ -41,5 +45,43 @@ if [ ! -z "$INPUT_SOFT_FAIL" ]; then
 fi
 
 echo "::add-matcher::checkov-problem-matcher.json"
-echo "running checkov on directory: $1"
-checkov -d $INPUT_DIRECTORY $CHECK_FLAG $SKIP_CHECK_FLAG $QUIET_FLAG $SOFT_FAIL_FLAG $FRAMEWORK_FLAG $EXTCHECK_DIRS_FLAG $EXTCHECK_REPOS_FLAG $OUTPUT_FLAG
+
+echo $(checkov --version )
```

Comment:
Why are you arbitrarily echoing the version here with no added context. If this was intended for debugging, it is best to remove it.